README.md

```diff
@@ -143,9 +143,13 @@ real-world use, add the `--release` flag to create an optimized build:
 ``` sh
 git clone https://github.com/servo/servo
 cd servo
-ANDROID_TOOLCHAIN=/path/to/toolchain ANDROID_NDK=/path/to/ndk PATH=$PATH:/path/to/toolchain/bin ./mach build --android
-cd ports/android
-ANDROID_SDK=/path/to/sdk make install
```

Comment:
Won't the `ANDROID_SDK` environment variable still need to be set? I don't think that `./mach` can figure that out itself